_pytest/terminal.py

```diff
@@ -355,7 +361,10 @@ def pytest_runtest_logfinish(self, nodeid):
                 self._write_progress_information_filling_space()
             else:
                 past_edge = self._tw.chars_on_current_line + self._PROGRESS_LENGTH + 1 >= self._screen_width
-                if past_edge:
+                width = self._tw.chars_on_current_line % self._screen_width + 9
```

Comment:
Again a magic constant, please use a named constant instead. 😁

Reply:
I see `9` should be `_MAX_LEN_LEFT - 1` if we add a constant for it, is that right?